test/golden/artman_golden_test.py

```diff
@@ -81,6 +81,7 @@ def generate_gapic_library(golden_dir, googleapis_dir, output_dir,
         '--local',
         '--root-dir', googleapis_dir,
         '--output-dir', output_dir,
+        '--user-config', "/artman/artman-user-config-in-docker.yaml",
```

Comment:
How will this file appear in the docker image?

Reply:
In the docker image, it would be at [/root/.artman/config.yaml](https://github.com/googleapis/artman/blob/e03351421771447e731771c04acb4aa3867c9a39/Dockerfile#L175)